src/globus_cli/commands/list_commands.py

```diff
@@ -43,11 +43,6 @@ def list_commands() -> None:
     """
     Prints the name and a short description of every command available in the globus
     cli. Commands are grouped by their parent commands,
```

Comment:
I think that the example could be updated, rather than deleting it.

If it's to stay deleted, though, this needs to convert to a period.

```suggestion
    cli. Commands are grouped by their parent commands.
```

Reply:
Let's make it a period! I forgot to mention this in the PR commentary: when I checked the command for another example, I found that the text was simply inaccurate!